pure_changelog.md

```diff
@@ -33,6 +33,7 @@
 * Fixed USB connection/disconnection detection
 * Fixed memory leaks in APN settings
 * Fixed windows flow regarding PUK requests after too many PIN mistakes
+* Fixed regression that pop-ups informing about PIN operations didn't apperar
```

Comment:
I wouldn't say that was a regression if it was not in the previous release. If so, then that's correct

Reply:
Checked - not really needed, thanks for pointing this out. I reverted the changelog modification.